### DISCUSSION TITLE
Multiple Alts

### DISCUSSION THREAD

**Discussion by Sh4rK157** [marioparaschiv/nitro-sniper#52] [Q&A] [answered]
How can I have more than one alt running?  everytime I try to use "," to split the alts tokens it gives me an error.

**Comment by DarkGD123** [comment, marked as answer]
In the alt part in `settings.json`
just do this

"Token1",
"token2",
"Token3",

**Reply by Sh4rK157** [reply]
https://imgur.com/a/CSBOz7R

can u help me?
It works when i have only  one alt, but when I have more than 2 it crashes.

**Reply by DarkGD123** [reply]
Remove the , in the last part

**Reply by DarkGD123** [reply]
Also i think the main token part does not need , so remove it
